eox_tenant/settings/common.py

```diff
@@ -36,12 +36,12 @@ def plugin_settings(settings):
     # Plugin settings.
```

Comment:
@andrey-canon there is a discussion in this [PR](https://github.com/eduNEXT/eox-core/pull/143) about not changing the defaults yet and wait for the official release of lilac. Maybe we should come to an agreement on this for the plugins

@mariajgrimaldi @morenol